packages/aws-cdk/lib/os.ts

```diff
@@ -23,7 +23,7 @@ export async function shell(command: string[]): Promise<string> {
 
     // Both write to stdout and collect
     child.stdout.on('data', chunk => {
-      process.stdout.write(chunk);
+      data(chunk);
```

Comment:
this probably no good. This used to just print chunks and now we are formatting chunks.

Reply:
For now revert and make backlog item to deal with this.

Only used in `cdk init` apparently to do some setting up of Dotnet apps

Reply:
Previously this line indiscriminately wrote data chunks from the child process into the output stream. These chunks are probably strings, although technically they can be a buffer. 

The chunk then gets passed through `formatMessageAndLog` and `log`, to end up on `ioHost.notify`.
While this is probably safe, the logging helpers as written may or may not format chunks. Going forward, we should have a more explicit API for data chunks.

Reply:
This is just to be defensive about unintentional changes.

Reply:
https://github.com/orgs/aws/projects/257/views/1?pane=issue&itemId=93573012&issue=aws%7Caws-cdk%7C32902